pkg/plugin/aggregation/update.go

```diff
@@ -163,26 +166,25 @@ func GetPatch(annotation string) map[string]interface{} {
 	}
 }
 
-// SetStatusPodName sets the sonobuoy master pod name based on it's label.
-func SetStatusPodName(client kubernetes.Interface, namespace string) {
-
+// GetStatusPodName gets the sonobuoy master pod name based on its label.
+func GetStatusPodName(client kubernetes.Interface, namespace string) (string, error) {
 	listOptions := metav1.ListOptions{
 		LabelSelector: StatusPodLabel,
 	}
 
 	podList, err := client.CoreV1().Pods(namespace).List(listOptions)
 	if err != nil {
-		logrus.Errorf("Error listing pods with label '%s': %s", StatusPodLabel, err)
-		return
+		return "", errors.Wrap(err, "Error listing pods with label %q")
```

Comment:
Should wrap with lower case phrases; only start with a capital when logging since you know it wont be further wrapped/prefixed.

Reply:
Thanks for pointing that out. I had taken this from the previous log line and forgot to update that.